prow/plugins/config.go

```diff
@@ -299,7 +299,7 @@ func (a Approve) ConsiderReviewState() bool {
 type Lgtm struct {
 	// Repos is either of the form org/repos or just org.
 	Repos []string `json:"repos,omitempty"`
-	// ReviewActsAsLgtm indicates that a Github review of "approve" or "request changes"
```

Comment:
Let's keep these out of other PRs just so 

1. you don't have to rebase your other PR all the time
2. more targeted diffs

Reply:
I'll revert these changes.